spec/fabricators/user_fabricator.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+Fabricator(:user) do
+  id { 1 }
+  provider { 'google_oauth2' }
```

Comment:
Could we keep it with `google`? 🙏

Reply:
@Tuubz It's the provider is being used to determine google OAuth2 from omniauth:
- https://github.com/zquestz/omniauth-google-oauth2#auth-hash

I'd prefer to keep it for consistency and easier to generate test data with Faker:

- https://github.com/faker-ruby/faker/blob/main/doc/default/omniauth.md